src/include/units/bits/basic_concepts.h

```diff
@@ -321,6 +321,44 @@ concept QuantityLike = detail::is_quantity_like<T>;
 template<typename T>
 concept QuantityPointLike = detail::is_quantity_point_like<T>;
 
+// PointOrigin
+namespace detail {
+
+template <typename, Unit>
+struct _origin_base;
+
+template <typename T>
+concept has_different_reference_origin_ =
```

Comment:
I'd prefer if this were affirmative. Like in the documentation above, you use "related" (like `quantity_kind`'s does).
```suggestion
concept related_reference_origin_ =
```